### DISCUSSION TITLE
parcel 2 scripts help

### DISCUSSION THREAD

**Discussion by moezbenrebah** [parcel-bundler/parcel#7654] [Q&A]
I used parcel-bundler in my project but some issues occurred so I decided to move to the next version I have some confusion about how to upgrade my scripts and even I go through the documentation I didn't figure it out. The point that I trying to reach is how to immigrate the below script to parcel 2:
```js
"watch:js": "parcel watch ./public/js/index.js --out-dir ./public/js --out-file bundle.js",
"build:js": "parcel build ./public/js/index.js --out-dir ./public/js --out-file bundle.js"
```

I'll be grateful for your help

**Comment by ashutoshofficial25** [comment]
I am using parcel 2.0.0
same error :

    "watch:js": "parcel watch ./public/js/index.js --out-dir ./public/js --out-file bundle.js",
    "build:js": "parcel watch ./public/js/index.js --out-dir ./public/js --out-file bundle.js"

error output:- 

parcel watch ./public/js/index.js --out-dir ./public/js --out-file bundle.js

error: unknown option '--out-dir'

please help..

**Reply by H0BB5** [reply]
@ashutoshofficial25 @moezbenrebah `--out-dir` has been changed to `--dist-dir`
There is no replacement for `--out-file` as it has been removed. Hope that helps
